primer-api/src/Primer/API.hs

```diff
@@ -985,6 +1030,10 @@ viewTreeExpr e0 = case e0 of
       , rightChild = Nothing
       }
   where
+    bindingNodeId = nodeId <> "V"
+    -- this ID will not clash with any others in the tree,
+    -- since node IDs in the input expression are unique,
+    -- and don't contain non-numerical characters
```

Comment:
This text already appeared twice in this file, and now appears four times. We could consider abstracting out this creation of special API-only IDs. Though these will disappear again if we do implement #75.